src/components/BlockchainList/PostsList.tsx

```diff
@@ -12,9 +12,7 @@ function BlockchainPostsListSuspended() {
 
   return (
     <>
-      {!selectedPosts.length ? (
-        <NoPosts />
-      ) : (
+      {selectedPosts.length ? (
```

Comment:
I guess it's `not necessary changes`. But I'm fine, to match code style

Reply:
Well, this is a little more optimized, because you don't need to do `!` + better readability